Bootstrap/Update.php

```diff
@@ -223,4 +224,18 @@ private function clearConfigCache()
             $cacheManager->clearConfigCache();
         }
     }
+
+    private function updateConnectAttribute(){
+        if (version_compare($this->version, '1.0.6', '<=')) {
+            $result = Shopware()->Db()->query("SELECT value FROM s_plugin_connect_config WHERE name = 'connectAttribute'");
+            $row = $result->fetch();
+            if ($row) {
+                Shopware()->Db()->query("
+                    UPDATE `s_articles_attributes` 
+                    SET `connect_reference` = `attr" . $row['value'] . "` 
```

Comment:
by default $row['value'] will be = 19. Can be between 1 and 20.
Why you need to set connect_reference value "attr19" for example?
I think we need to fetch which attribute is used right now, 19 for example, then select value from attr19 and insert it in connect_reference column.

connect_reference contains sourceId only for imported products. So this query must be changed.